src/analyses/ai.cpp

```diff
@@ -499,10 +499,11 @@ void ai_baset::sequential_fixedpoint(
   const goto_functionst &goto_functions,
   const namespacet &ns)
 {
-  // do each function at least once
+  goto_functionst::function_mapt::const_iterator
+    f_it=goto_functions.function_map.find(goto_functions.entry_point());
 
-  forall_goto_functions(it, goto_functions)
-    fixedpoint(it->second.body, goto_functions, ns);
+  if(f_it!=goto_functions.function_map.end())
+    fixedpoint(f_it->second.body, goto_functions, ns);
```

Comment:
Ah right. This is a general framework fix as it does not make sense to run the fixpoint on other functions when we don't set the entry state (as discussed with @peterschrammel). Should I make a separate commit or pull request for that?